frameworks/helloworld/src/main/java/com/mesosphere/sdk/helloworld/scheduler/Main.java

```diff
@@ -49,6 +49,15 @@ public static void main(String[] args) throws Exception {
                         schedulerConfig,
                         yamlSpecFile.getParentFile());
                 break;
+            case MULTI_REGION:
+                yamlSpecFile = new File(args[0]);
+                serviceSpec = DefaultServiceSpec
+                        .newGenerator(yamlSpecFile, SchedulerConfig.fromEnv())
+                        .build();
+                builder = DefaultScheduler.newBuilder(serviceSpec, SchedulerConfig.fromEnv())
+                        .withSingleRegionConstraint();
```

Comment:
It seems a bit strange to have the `MULTI_REGION` case call a method with the name of `withSingleRegionConstraint()`.

Reply:
`MULTI_REGION` describes the testing scenario -- whether in a multi-region cluster or not, we _always_ mandate that the scheduler run its tasks in a single region, so the point here is that `withSingleRegionConstraint` really only makes sense to assert when in a multi-region environment in the first place.